solc/CommandLineInterface.cpp

```diff
@@ -1097,6 +1097,21 @@ void CommandLineInterface::outputCompilationResults()
 {
 	solAssert(m_options.input.mode == InputMode::Compiler || m_options.input.mode == InputMode::CompilerWithASTImport, "");
 
+	std::set<std::string> uncompiledContracts;
```

Comment:
Our convention is to use `using namespace std;` in .cpp files and drop the `std::` prefix there.
```suggestion
	set<string> uncompiledContracts;
```